server.ts

```diff
@@ -74,9 +78,17 @@ app.use(
     rootValue: root,
     graphiql: true,
     formatError
-  })
+  }),
+  graphqlExpress({ schema })
 )
 
+//Is this somehow different from the code above? Is this necessary? Maybe we just add the subscriptionsEndpoint line
+//to the code above?
+app.use('/graphiql', graphiqlExpress({
+  endpointURL: 'graphql',
+  subscribtionsEndpoint: `ws://localhost:${port}/subscriptions`
```

Comment:
Ya, I believe you can move line 89 inside the body of graphqlHTTP above line 81. I'm not sure what the graphqlExpress method is doing for us here

Reply:
That's how they do it in this example: https://github.com/graphql/express-graphql/issues/390